### ISSUE TITLE
PLA or ABS to make these?

### ISSUE THREAD

**Issue by DonnieNarco** [brentjackson/OpenIntubator#10]
Hello, I would like to contribute but I am not sure if I can make these with PLA or if I would need to use ABS or other types of filament. I currently can only print PLA. 
Will that suffice for this application? Thank you.

**Comment by wbphelps** [comment]
PLA or CF PLA would be more than adequate, and far easier to print than ABS. You might also try PETG. I haven't printed ABS in at least 2 years now.

**Comment by kyranf** [comment]
Before we can actually conclude on this, we need to know what the requirements are. Temperature, humidity, water/air tightness, and sterilisability. It's hard to sterilize PLA with hot steam/gas, if it melts at 100 degrees.  ABS or PETG which have much better temperature handling capability might be best. 

ABS with a acetone vapour bath to smooth the surfaces and make them unable to be used by bacteria as growing surfaces is important for the health/hygiene.

**Comment by vinnie90** [comment]
I agree. It will be hard to keep anything clean. Might be a one time use thing

**Comment by wbphelps** [comment]
If you're talkiing about sterilization, forget it, even PETG won't stand up to that. I have personal experience with this - I printed some tool holders for a dentist friend using PETG, and the sterilizer destroyed them.

**Comment by erosmarcello** [comment]
A friend of mine recommended an inflatable infrastructure - they have them for modding painting vehicles. I will attach links.
